tools/schemacode/src/bidsschematools/render/tsv.py

```diff
@@ -1,15 +1,56 @@
+from __future__ import annotations
+
 import io
+import re
 
 import pandas as pd
 from markdown_it import MarkdownIt
 from tabulate import tabulate
 
 
-def fence(source: str, language: str, css_class: str, options: dict, md, **kwargs) -> str:
+def fence(
+    source: str,
+    language: str,
+    css_class: str,
+    options: dict,
+    md,
+    classes=None,
+    id_value="",
+    attrs: dict | None = None,
+    **kwargs,
+) -> str:
+    attrs = attrs or {}
+    linenums = attrs.get("linenums", "0") == "1"
+
+    if language == "tsvgz" and "header" not in attrs:
+        attrs["noheader"] = True
+
+    classes.insert(0, "tsv-table")
+    classes.insert(1, "index" if linenums else "noindex")
```

Comment:
```suggestion
    if classes is not None:
        classes.insert(0, "tsv-table")
        classes.insert(1, "index" if linenums else "noindex")
```

Reply:
Classes should not be none, but we could rather:
```suggestion
    classes = classes or []
    classes.insert(0, "tsv-table")
    classes.insert(1, "index" if linenums else "noindex")
```